mypy/reachability.py

```diff
@@ -166,9 +165,14 @@ def consider_sys_platform(expr: Expression, platform: str) -> int:
         op = expr.operators[0]
         if op not in ('==', '!='):
             return TRUTH_VALUE_UNKNOWN
-        if not is_sys_attr(expr.operands[0], 'platform'):
-            return TRUTH_VALUE_UNKNOWN
-        right = expr.operands[1]
+        # check if either platform or platform_system is being used
+        if platform_system:
```

Comment:
Right now you are saying the truth value is unknown if the expression is not a platform attribute, but you don't want to do that. Both `platform` and `platform_system` should be truthy, since they will both be set in options.